src/processors/InputProcessor.ts

```diff
@@ -12,6 +13,7 @@ export class InputProcessor {
 
   constructor() {
     this.setProcessor('asyncapi', new AsyncAPIInputProcessor()); 
+    this.setProcessor('swagger', new SwaggerInputProcessor()); 
```

Comment:
I wonder if we need to create a swagger processor by default. This is such a rare case that the user should create it only if he/she wants to - why is this so important to me? For this same problem like we have with `fs` package in another PR - for tree-shaking. Whenever someone wants to use the generation just for AsyncAPI or plain JSON Schema, they will add dependencies for swagger to the resulting output bundle (case for Modelina playground on the website and in the future for the Studio) - which isn't necessary and only add additional, not used code to bundle. What do you think?

Reply:
Hmm, I definitely see the use case, however, when would you want to see this being addressed, now, or later?

Cause I assume we would need to split up some things for this to work, or is it painless? 🤔

Reply:
We can later. Probably when we will integrate the Modelina in the Studio (not sure when 😆 )